### PR TITLE
Added generic Card UI comp. and MealItem comp.

### PR DESCRIPTION
**Pull request by darrenhuang1211**
Added a reusable Card component. Added a MealItem component to render individual items on the list of meals. Added a skeleton of the form in each MealItem as well.

### CODE REVIEW

File: src/components/Meals/MealItem.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+   .price {
+      margin-top: 0.25rem;
+      font-weight: bold;
+      color: #ad5502;
+      font-size: 1.25rem;
+   }
+`;
+
+const MealItem = props => {
```
Comment (oliverlin) [inline src/components/Meals/MealItem.js:26]: can list out all props explicitly here so we know what props are being used.
```es6
const MealItem = ({ price, name, description }) => {
```

File: src/components/Meals/MealItem.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+               <label>Amount</label>
+               <input type="number"></input>
+               <button type="submit">+ Add</button>
+            </form>
+         </div>
+      </StyledMealItem>
+   );
+};
+
+export default MealItem;
```
Comment (oliverlin) [inline src/components/Meals/MealItem.js:47]: can use PropTypes package to define the props of this component. So we know what props are available for this component. https://www.npmjs.com/package/prop-types

File: src/components/Meals/AvailableMeals.js [hunk 3]
```diff
@@ -54,16 +56,23 @@ const StyledMealsList = styled.section`
...
-         <ul>
-            {mealsList.map(mealToElement)}
-         </ul>
+      <StyledMealsList className={props.className}>
+         <Card>
+            <ul>
+               {mealsList.map(mealToElement)}
+            </ul>
+         </Card>
       </StyledMealsList>
```
Comment (oliverlin) [inline src/components/Meals/AvailableMeals.js:76]: `mealToElement` function does not really do anything. 
Can change it to following to simplify it
```es6

<StyledMealsList className={props.className}>
  <Card>
    <ul>
      {mealsList.map((meal) => (
        <MealItem 
          key={meal.id} 
          name={meal.name}
          description={meal.description}
          price={meal.price} />
      ))}
    </ul>
  </Card>
</StyledMealsList>
```

File: src/components/Meals/MealItem.js (hunk 1 above)
Comment (darrenhuang1211) [reply src/components/Meals/MealItem.js:26]: Thanks for the tip! If I use this, will I still receive "children" and "className" properties?

File: src/components/Meals/AvailableMeals.js (hunk 3 above)
Comment (darrenhuang1211) [reply src/components/Meals/AvailableMeals.js:76]: Got it. I thought to make a separate function so it's easier to read

**Pull request merged**